File: src/main/java/com/optica/controllers/MappingConstants.java
package com.optica.controllers;

public interface MappingConstants {

	String INDEX_PATH = "/";
	
	String LOGIN_PATH = "/login";
	
	String REGISTRATION_PATH = "/registration";

	String ABOUT_PATH = "/about";

	String CONTACT_PATH = "/contact";

	String TEAM_PATH = "/team";

	String BLOG_PATH = "/blog";

	String PEOPLE_PATH = "/people";

	String JOB_SEARCH_PATH = "/search/{keyword:.*}/{city:.*}";

	String JOB_DETAIL_PATH = "/detail/{title:.*}/{id:.*}";

	String JOB_OFFER_NEW = "/job-offer/";

}

File: src/main/kotlin/com/optica/controllers/MappingConstants.kt
package com.optica.controllers

interface MappingConstants {
    companion object {

        const val INDEX_PATH = "/"

        const val LOGIN_PATH = "/login"

        const val REGISTRATION_PATH = "/registration"

        const val ABOUT_PATH = "/about"

        const val CONTACT_PATH = "/contact"

        const val TEAM_PATH = "/team"

        const val BLOG_PATH = "/blog"

        const val PEOPLE_PATH = "/people"

        const val JOB_SEARCH_PATH = "/search/{keyword:.*}/{city:.*}"

        const val JOB_DETAIL_PATH = "/detail/{title:.*}/{id:.*}"

        const val JOB_OFFER_NEW = "/job-offer/"
    }

}
